api.py

```diff
@@ -533,10 +567,9 @@ def ingest_entities(dataset_id, data, **kwargs):
         Requests that result in an error raise an exception with information about the failure. See :py:func:`requests.Response.raise_for_status` for more information.
     """
 
-    if isinstance(data, list):
-        data = {'entities': data}
+    entity_set = convert_entities_to_entity_set(entity_list)
```

Comment:
Wondering if we should do some checks here.  For instance, require no more than one entity per ID, since they will just overwrite one another.  Are there others?